00-intro/intro.md

```diff
@@ -15,39 +15,33 @@ links-as-notes: true
 
 ## Overview
 
-\Gls{NeoFS} is a decentralized distributed object storage system integrated with
-the [Neo Blockchain](https://neo.org).
+\Gls{NeoFS} is a decentralized distributed object storage system integrated with [Neo Blockchain](https://neo.org).
 
-\Glspl{Node} are organized in peer-to-peer network that takes care of storing
-and distributing user's data. Any Neo user may participate in the network and
-get paid for providing storage resources to other users or store his data in
-NeoFS and pay a competitive price for it.
+We store and distribute users' data across a peer-to-peer network of \Glspl{Node}. Whether a business or an individual, any Neo user may join the network and get paid for providing storage resources to others or pay a competitive price and enjoy NeoFS storage solution.
 
-Users can reliably store object data in the NeoFS network and have a transparent
-data placement process due to decentralized architecture and flexible storage
-policies. Under the users we mean both individuals and enterprises. Each
-\Gls{Node} is responsible for executing the storage policies that the users
-select for geographical location, reliability level, number of nodes, type of
-disks, capacity, etc. Thus, NeoFS gives full control over data to users.
+The decentralized architecture and flexible storage
+policies allow users to reliably store object data in the NeoFS network. Each \Gls{Node} is responsible for executing the storage policies selected by the users
+regarding the geographical location, reliability level, number of nodes, type of
+disks, capacity, etc. Thus, NeoFS enables a transparent
+data placement process and gives full control over data to the users.
 
 Deep [Neo Blockchain](https://neo.org) integration allows NeoFS to be used by
 \glspl{dApp} directly from
 [NeoVM](https://docs.neo.org/docs/en-us/basic/technology/neovm.html) on the
 [Smart
 Contract](https://docs.neo.org/docs/en-us/basic/technology/neocontract.html)
-code level. This way dApps are not limited to on-chain storage and can
+code level. As a result, dApps are not limited to on-chain storage and one can
 manipulate large amounts of data without paying a prohibitive price.
 
-NeoFS has native [gRPC](https://grpc.io) \Gls{api} and popular protocol gates
+NeoFS uses native [gRPC](https://grpc.io) \Gls{api} and popular protocols
 such as [AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html),
 [HTTP](https://wikipedia.org/wiki/Hypertext_Transfer_Protocol),
-[FUSE](https://wikipedia.org/wiki/Filesystem_in_Userspace) and
-[sFTP](https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol) allowing
-developers to easily integrate their applications without rewriting their code.
+[FUSE](https://wikipedia.org/wiki/Filesystem_in_Userspace), and
+[sFTP](https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol), which allows
+developers to easily integrate their applications without rewriting their codes.
```

Comment:
I'm not sure if code as a program's source code can be used in plural form. Can we check this?

Reply:
I meant to emphasize that there are various apps using various types (may I say so?) of codes. But I'll look it up.

Reply:
I've found enough examples of "source codes of applications" (e.g. GitHub: "You can find source codes of applications or custom animations / UI.")